CONTRIBUTORS

```diff
@@ -18,6 +18,8 @@ SIG-archinfra:
     - kevinch-nv
     - winnietsang
     - chinhuang007
+    - jcwchen
+    - vinitra
```

Comment:
both Jacky and Vinitra are actively participating in resolving onnx issues, build errors and improving CI pipelines, checker, shape inference etc...